### PR TITLE
Add proposal to initialize the configuration needed to enable yearly reward rate reduction

### PR DESCRIPTION
**Pull request by movekevin**

### CODE REVIEW

File: sources/2023-09-19-initialize-reward-rate-configuration/initialize.move [hunk 1]
```diff
@@ -0,0 +1,20 @@
+script {
+    use aptos_framework::aptos_governance;
+    use aptos_framework::staking_config;
+    use aptos_std::fixed_point64;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(proposal_id, @0000000000000000000000000000000000000000000000000000000000000001, vector::empty<u8>());
+        let (curr_rewards_nominator, curr_rewards_denominator) = staking_config::get_reward_rate(&staking_config::get());
```
Comment (junkil-park) [inline sources/2023-09-19-initialize-reward-rate-configuration/initialize.move:8]: It seems that `staking_config::get_reward_rate` does not exist on the head of `main`. Am I missing something? https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/staking_contract.move

File: sources/2023-09-19-initialize-reward-rate-configuration/initialize.move [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        staking_config::initialize_rewards(
```
Comment (junkil-park) [inline sources/2023-09-19-initialize-reward-rate-configuration/initialize.move:9]: `staking_config::initialize_rewards` doesn't seem to exist either.

File: sources/2023-09-19-initialize-reward-rate-configuration/initialize.move (hunk 1 above)
Comment (sherry-x) [reply sources/2023-09-19-initialize-reward-rate-configuration/initialize.move:8]: It's in the staking_config.move

File: sources/2023-09-19-initialize-reward-rate-configuration/initialize.move (hunk 2 above)
Comment (sherry-x) [reply sources/2023-09-19-initialize-reward-rate-configuration/initialize.move:9]: same as above

**Review by sherry-x — APPROVED** [review]

File: sources/2023-09-19-initialize-reward-rate-configuration/initialize.move (hunk 1 above)
Comment (junkil-park) [reply sources/2023-09-19-initialize-reward-rate-configuration/initialize.move:8]: Ah, I misread `staking_config` as `staking_contract`. Thank you!

Comment (junkil-park) [inline sources/2023-09-19-initialize-reward-rate-configuration/initialize.move:8]: minor comment: "nominator" --> "numerator"

**Review by junkil-park — APPROVED** [review]

**Pull request merged**